pkg/otel/common/schema/builder/string.go

```diff
@@ -48,13 +48,6 @@ func (b *StringBuilder) AppendNull() {
 
 func (b *StringBuilder) Append(value string) {
 	if b.builder != nil {
-		// This is a workaround for dictionaries that do not support empty
-		// strings.
-		if value == "" {
-			b.builder.AppendNull()
-			return
-		}
-
```

Comment:
I believe callers could use `AppendNonEmpty` instead to have empty string appended as null

Reply:
Thanks, makes sense.